client/styles/components/_forms.scss

```diff
@@ -52,7 +52,7 @@
 
 .form__input {
   width: 100%;
-  min-width: #{360 / $base-font-size}rem;
+  min-width: 10%;
```

Comment:
If the `width` is `100%` then the `min-width` of `10%` will never be used.  The `10%` will never be greater than the `100%`.  So this doesn't make a ton of sense.  BUT....it actually is an improvement over what we have now!  What you've discovered is that the current `min-width: 360px` (`min-width: #{360 / $base-font-size}rem;`) is not good as it causes horizontal overflow when the screen is narrower than ~390px.

Instead of changing it to `min-width: 10%;`, let's delete that line.